invenio_records_rest/views.py

```diff
@@ -832,6 +831,12 @@ def get(self, **kwargs):
 
         # Execute search
         response = s.execute_suggest().to_dict()
+        if ES_VERSION[0] == 2:
```

Comment:
For ES5 we should have an option for source filtering so that we have control over what is exposed in the suggestion endpoint.

Reply:
If we pass the whole "_source" we could pass it through the serializer.